### ISSUE TITLE
AttributeError on installing game from commandline (`lutris -i installer.yml`)

### ISSUE THREAD

**Issue by Freso** [lutris/lutris#69]
```
freso@kotake> LANG=C lutris -v -d -i the-dark-mod-201.yml

** (lutris:4848): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-bZPm0Xl65r: Connection refused
INFO     2014-07-07 17:44:23,863 [lutris]:Installing 
Traceback (most recent call last):
  File "/usr/bin/lutris", line 126, in <module>
    InstallerDialog(options.installer_file or game_slug)
  File "/usr/lib/python2.7/site-packages/lutris/installer.py", line 697, in __init__
    self.launch_install(0)
  File "/usr/lib/python2.7/site-packages/lutris/installer.py", line 703, in launch_install
    self.interpreter = ScriptInterpreter(script, self)
  File "/usr/lib/python2.7/site-packages/lutris/installer.py", line 100, in __init__
    if not self.is_valid():
  File "/usr/lib/python2.7/site-packages/lutris/installer.py", line 138, in is_valid
    if not self.script.get(field):
AttributeError: 'str' object has no attribute 'get'
```

(Using the latest Git master via the lutris-git ArchLinux package, as of this report commit 11c43f9.)
